src/icons/avatars/UnknownUserAvatar.tsx

```diff
@@ -14,8 +14,6 @@ class UnknownUserAvatar extends React.PureComponent<TwoTonedIcon> {
         width: 28,
     };
 
-    idPrefix = `${uniqueId(ICON_CLASS)}-`;
```

Comment:
Should we be removing this unique ID? Do we know what depends on it?

Reply:
This is only used in the definition and reference of the circle in the old avatar svg, so it is safe to remove.

Reply:
Are you sure that the ID of the icon isn't referenced by other code indirectly, though? I don't mean the variable reference. What was the original purpose of having an ID on the icon?